src/pepr/operator/crd/register.ts

```diff
@@ -6,12 +6,50 @@
 import { K8s, kind } from "pepr";
 
 import { Component, setupLogger } from "../../logger";
+import { v1alpha1 as clusterConfig } from "./sources/cluster-config/v1alpha1";
 import { v1alpha1 as exemption } from "./sources/exemption/v1alpha1";
 import { v1alpha1 as pkg } from "./sources/package/v1alpha1";
 
 // configure subproject logger
 const log = setupLogger(Component.OPERATOR_CRD);
 
+export async function registerClusterConfig() {
+  // Register the ClusterConfig CRD
+  if (process.env.PEPR_WATCH_MODE || process.env.PEPR_MODE === "dev") {
```

Comment:
```suggestion
  if (process.env.PEPR_WATCH_MODE === "true" || process.env.PEPR_MODE === "dev") {
```

I think? Unless I'm missing a reason this is applied from all pods.

Reply:
Revisiting this after reviewing the rest, since this is only called by the task maybe it's not a big issue?